python/README.rst

```diff
@@ -0,0 +1,20 @@
+Basix Python interface
+========================
+
+This document explains how to install the Basix Python interface. Note that
```

Comment:
I would add a note along the lines that regular users should use the top-level install.